### DISCUSSION TITLE
add safe.directory

### DISCUSSION THREAD

**Discussion by eddhyne** [community/community#60398] [Programming Help]
When I run git status I get
(.venv) PS Z:\Django Projects\Ed's_helloworld> git status
fatal: detected dubious ownership in repository at '//ISICSFLSYN01/ISICSFLSYN01/Django Projects/Ed's_helloworld'
'//ISICSFLSYN01/ISICSFLSYN01/Django Projects/Ed's_helloworld' is owned by:
        'S-1-5-21-928949504-4148710699-525549864-3058'
but the current user is:
        'S-1-5-21-2909440574-4242095837-2197871977-1001'
To add an exception for this directory, call:

        git config --global --add safe.directory '%(prefix)///ISICSFLSYN01/ISICSFLSYN01/Django Projects/Ed'\''s_helloworld'
>> what do I put in here?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @eddhyne ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Reply by ShoaibMirzax** [reply]
git config --global --add safe.directory, next what you write I don't understand. I faced this problem and in every project used this command, is there any way to fix this?
